### DISCUSSION TITLE
Does it work with react-native for windows+macos?

### DISCUSSION THREAD

**Discussion by gabimoncha** [tamagui/tamagui#123] [Q&A]
I'm considering using tamagui to replace our current styled-system implementation, but don't know if it supports [react-native Windows + macOS](https://microsoft.github.io/react-native-windows/)

**Comment by natew** [comment]
Would love to support this, but as now it doesn't.

**Reply by redbar0n** [reply]
What is Tamagui currently missing? Maybe someone would be willing to pick up the work for integrating it, it you wrote a bit about what would be needed. Now, with the release candidate out, it seems like a good point to start distributing the work.
